superset/commands/chart/export.py

```diff
@@ -85,3 +102,12 @@ def _export(
 
         if model.table and export_related:
             yield from ExportDatasetsCommand([model.table.id]).run()
+
+        # Check if the calling class is ExportDashboardCommands
+        if (
+            export_related
+            and ExportChartsCommand._include_tags
+            and feature_flag_manager.is_feature_enabled("TAGGING_SYSTEM")
+        ):
+            chart_id = model.id
+            yield from ExportTagsCommand().export(chart_ids=[chart_id])
```

Comment:
Why only include tags for dashboard exports? Why not always include tags here?

Reply:
I think this logic is in place to prevent calling `ExportTagsCommand.export()` multiple times. It seems that if it's a dashboard export, then we don't export tags at the chart export level, and then at the dashboard level:
``` python
ExportTagsCommand.export( dashboard_ids=dashboard_ids, chart_ids=chart_ids )
```
When exporting charts only, then we export tags at the chart level.